ingestion/src/metadata/ingestion/api/workflow.py

```diff
@@ -147,6 +154,17 @@ def create(cls, config_dict: dict) -> "Workflow":
         config = parse_workflow_config_gracefully(config_dict)
         return cls(config)
 
+    # TODO TOTEST
+    def _get_source_dir(self, type):
```

Comment:
there's something like this already present when validating sources

Reply:
https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/ingestion/api/parser.py might be useful

Reply:
also, what is the `type` of type?? Maybe we should add some type hints 😛